src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+
+    This service account will be referred to as the _provisioning service account_ in the remainder of this page.
+
+1. Create a second service account and grant it the Cluster Admin role. This service account will be referred to as the folder-management service account. When prompted, create an API key for the service account and make a note of it. In the service account's **Details** page, make a note of the service account's ID, which is the portion of the URL after the last `/` character.
+
+    This service account will be referred to as the _folder-management service account_ in the remainder of this page.
+
+### Grant the `FOLDER_ADMIN` role to the service account
+
+Folders inherit roles granted at the organization scope, and descendent folders inherit roles granted higher in the hierarchy.
+
+During Limited Access, it is not possible to use the {{ site.data.products.db }} Console to grant the Folder Admin or Folder Mover roles.
+Instead, use the CockroachDB {{ site.data.products.db }} API and the provisioning service account you created in [Gather required information](#gather-required-information) to grant the folder-management service account the `FOLDER_ADMIN` role at the level of the organization.
+
+1. To grant the role at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<folder-management-service-account-id>/ORGANIZATION/<organization_id>/FOLDER_ADMIN' \
+           --header 'Authorization: Bearer <provisioning_service_account_api-key>'
+    ~~~
+
+    Replace:
+    - `<folder-management-service-account-id>`: the ID of the user or service account being granted the role.
+    - `<organization_id>`:your organization's ID.
+    - `<provisioning_service_account_api-key>`: The API key of a service account with the Org Admininstrator role.
+
+    After your desired folder structure is in place, you could revoke the role from the organization and grant it on individual folders instead. A recommended security practice is to limit the users or service accounts with `FOLDER_ADMIN` at the level of the organization. When you [create a folder](#create-a-folder-and-grant-access-to-it), you can grant the `FOLDER_ADMIN` role at the level of the folder.
+
+1. To revoke the role at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request DELETE \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<user-id>/ORGANIZATION/<organization_id>/FOLDER_ADMIN' \
+           --header 'Authorization: <api-key>'
+    ~~~
+
+    Replace:
```

Comment:
I would suggest the following here:

- `<service-account-id>`: the service account ID (the role grantee)
- `<organization_id>`:your organization's ID.
- `<api-key>`: the role granter's API Key (not the API Key of the role grantee)

Reply:
Done